### ISSUE TITLE
Error occur while running icon to archway node

### ISSUE THREAD

**Issue by Prathiksha-Nataraja** [HugoByte/DIVE#160]
## Overview
While running icon to archway node with the ibc bridge enabled, getting the error.

## Steps to Reproduce
1. kurtosis run . '{"action":"setup_relay","relay":{"name":"ibc","links": {"src": "icon", "dst": "archway"},"bridge":"false"}}' --enclave ibc

## Expected Behavior
IBC bridge Setup Completed between icon and archway

## Stack trace
There was an error executing Starlark code 
An error occurred executing instruction (number 45) at github.com/hugobyte/dive/services/jvm/icon/src/node-setup/setup_icon_node.star[16:23]:
  wait(service_name="icon-node-0xacbc4e", recipe=PostHttpRequestRecipe(port_id="rpc", endpoint="/api/v3/icon_dex", body="{ \"jsonrpc\": \"2.0\", \"id\": 1, \"method\": \"icx_call\", \"params\": { \"to\": \"cx0000000000000000000000000000000000000000\", \"dataType\": \"call\", \"data\": { \"method\": \"getMainPReps\", \"params\": {  } } } }", content_type="application/json", extract={"preps": ".result.preps"}), field="code", assertion="==", target_value=200)
  Caused by: An error occurred setting value 'map[body:"{\"jsonrpc\":\"2.0\",\"result\":{\"blockHeight\":\"0x18\",\"preps\":[],\"totalDelegated\":\"0x0\",\"totalPower\":\"0x0\"},\"id\":1}\n" code:+200 extract.preps:[]]' using key UUID '6219796437b74f8c9eaf84fae1e1b5cc' in the runtime value store
  Caused by: An error occurred saving value 'map[body:"{\"jsonrpc\":\"2.0\",\"result\":{\"blockHeight\":\"0x18\",\"preps\":[],\"totalDelegated\":\"0x0\",\"totalPower\":\"0x0\"},\"id\":1}\n" code:+200 extract.preps:[]]' using UUID key '6219796437b74f8c9eaf84fae1e1b5cc' into the recipe result repository
  Caused by: An error occurred while saving recipe result value 'map[body:"{\"jsonrpc\":\"2.0\",\"result\":{\"blockHeight\":\"0x18\",\"preps\":[],\"totalDelegated\":\"0x0\",\"totalPower\":\"0x0\"},\"id\":1}\n" code:+200 extract.preps:[]]' with UUID '6219796437b74f8c9eaf84fae1e1b5cc' into the enclave db
  Caused by: Unexpected comparable type on recipe result repository, only 'starlark.String and slartark.Int' are allowed but '[]' was received.

Error encountered running Starlark code.

**Comment by shreyasbhat0** [comment]
@Prathiksha-Nataraja please run command with bridge field
```
kurtosis run . '{"action":"setup_relay","relay":{"name":"ibc","links": {"src": "icon", "dst": "archway"}}}' --enclave ibc
```

**Comment by shreyasbhat0** [comment]
Closed with PR #164
